src/Pact/Analyze/Types/Eval.hs

```diff
@@ -195,7 +198,8 @@ data AnalyzeState
 
 data AnalysisResult
   = AnalysisResult
-    { _arProposition   :: SBV Bool
+    { _arEvalFailure   :: SymbolicSuccess
```

Comment:
this is confusing

Reply:
other option here vs newtype-wrapping or type alias, could take the same approach as with https://github.com/kadena-io/pact/blob/d4f96422d52464aa5a20b58d4599d13fcc161b76/src/Pact/Analyze/Types/Shared.hs#L136-L149

not sure if worth it though